### PR TITLE
EIPIP meeting notes 36

### PR DESCRIPTION
**Pull request by dschirmerii**
Updated notes and readme file

### CODE REVIEW

File: All EIPIP Meetings/meeting 036.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
+# EIPIP Meeting 36 Notes
+### Meeting Date/Time: Wednesday 2021/06/30 at 15:00 UTC
+### Meeting Duration: 50 mins
+### [GitHub Agenda](https://github.com/ethereum-cat-herders/EIPIP/issues/75)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=wt7EB8MujgA&ab_channel=EthereumCatHerders)
+### Moderator: Pooja Ranjan
+### Notes: David Schirmer
+## Action items
```
Comment (poojaranjan) [inline All EIPIP Meetings/meeting 036.md:8]:
```suggestion
## Agenda items

```

File: All EIPIP Meetings/meeting 036.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+| Decision Item | Decision Made |
+| ------------- | ------------|
+| **1**   | TBD, more time is needed to find a solution  |     
+| **2**   | Short video will be made showing the basics of editing |  
+| **3**   | Coding finished, formatting meeting will be held at a later date. |  
+| **4**   | Not finished, need more time. |  
+| **5**   | Time limitation will outsource to community for issues. | 
+
+## Meeting Notes
+**Pooja Ranjan**( Evolving EIP process)
```
Comment (poojaranjan) [inline All EIPIP Meetings/meeting 036.md:26]:
```suggestion
### Evolving EIP process
**Pooja Ranjan**

```

File: All EIPIP Meetings/meeting 036.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+**Micah**
+ I think that one is actually you're talking about ep8 right. I think it was needed before homestead so I don't think it was actually included with homestead it was just every client needs to implement this before homestead goes live.
+ 
+**Pooja**
+ I know I think I saw it in the list of Proposals.
+ 
+**Micah**
+ did it actually you know did it actually go live with homestead it is there in the meta eip for Homestead okay that was before my time so I’ll take your word.
+ 
+**Pooja**(new eip editors-31:48)
```
Comment (poojaranjan) [inline All EIPIP Meetings/meeting 036.md:193]:
```suggestion
### New eip editors
**Pooja** (31:48)

```

File: All EIPIP Meetings/meeting 036.md [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+**Brent**
+Yeah that's a time issue for me I need to fork myself but I haven't got it's still on my list of things to do but haven't done anything recently so gotta push it again.
+ 
+**Pooja**
+ right uh like uh do you want me to keep it on recording meeting or should we take it off and one you have.
+ 
+**Brent**
+ yeah  let's try it at least one more meeting and i'll try and get something before the next meeting.
+ 
+**Pooja**(review action items-44:30)
```
Comment (poojaranjan) [inline All EIPIP Meetings/meeting 036.md:314]:
```suggestion
### Review action items
**Pooja** (44:30)

```

**Review by poojaranjan — COMMENTED** [review]
Overall looks good. Small edits suggested. Please add the Attendees list and the date for the next meeting to the notes. You may follow the format from any of the earlier meeting notes. 
Thank you!

File: All EIPIP Meetings/meeting 036.md (hunk 4 above)
Comment (dschirmerii) [reply All EIPIP Meetings/meeting 036.md:314]: Okay no worries I am in class but I will take care of it right after.

**Pull request merged**